packtools/sps/validation/utils.py

```diff
@@ -22,3 +22,11 @@ def format_response(
                 'message': f'Got {obtained}, expected {expected}',
                 'advice': None if is_valid else advice
             }
+
+
+ALTERNATIVES = {
+    "table-wrap": ["graphic", "table", "inline-graphic", "media", "supplementary-material"],
+    "fig": ["graphic", "inline-graphic", "media", "supplementary-material"],
+    "inline-formula": ["inline-graphic", "mml:math", "tex-math"],
+    "disp-formula": ["mml:math", "tex-math"],
+}
```

Comment:
@Rossi-Luciano ok ser for um valor padrão para um parâmetro de configuração. Minha sugestão é que seja passado por parâmetro

Reply:
OK, @robertatakenaka